custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -16,7 +16,8 @@
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'leaflet/dist/leaflet.css' %}">
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'angular-busy/dist/angular-busy.min.css' %}"/>
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'perfect-scrollbar/css/perfect-scrollbar.min.css' %}"/>
-        <link type="text/css" rel="stylesheet" media="all" href="{% static 'css/icds_dashboard.css' %}"/>
+        <link type="text/less" rel="stylesheet" media="all" href="{% static 'css/icds_dashboard.less' %}"/>
+        <link type="text/less" rel="stylesheet" media="all" href="{% static 'css/fact_sheets.less' %}"/>
```

Comment:
was thinking that we could opportunistically start refactoring styles into smaller files as we make changes, though also open to the idea that this makes things worse without some initial effort at organization (if we decide the latter I'll just move these changes into `icds_dashboard.less` for now).

Reply:
I'm in favor of this - I'm definitely glad to see fewer inline styles, and also generally like having more small, well-named files.